src/Jackett/Indexers/NxtGn.cs

```diff
@@ -0,0 +1,185 @@
+        }
+
+        public async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var breakWhile = false;
+            var page = 0;
+            while (true)
+            {
+                string episodeSearchUrl;
+                if (string.IsNullOrEmpty(query.GetQueryString()))
+                {
+                    episodeSearchUrl = SearchUrl + "?page=" + page;
+                    breakWhile = true;
+                }
+                else
+                {
+                    var cats = MapTorznabCapsToTrackers(query);
+                    var catsUrlPart = string.Join("&", cats.Select(c => $"c{c}=1"));
+                    episodeSearchUrl = string.Format("{0}?search={1}&cat=0&incldead=0&{2}&page={3}", SearchUrl, HttpUtility.UrlEncode(query.GetQueryString()), catsUrlPart, page);
+                }
+                page++;
+                var results = await RequestStringWithCookiesAndRetry(episodeSearchUrl);
+                try
+                {
+                    CQ dom = results.Content;
+
+                    var rows = dom["#torrent-table-wrapper > div"];
+
+                    foreach (var row in rows.Skip(1))
+                    {
+                        var release = new ReleaseInfo();
+
+                        var qRow = row.Cq();
+                        var qLink = qRow.Find("#torrent-udgivelse2-users > a").First();
+                        var qDesc = qRow.Find("#torrent-udgivelse2-users > p").FirstOrDefault();
+
+                        var torrentUrl = qLink.Attr("href");
+                        var torrentId = torrentUrl.Substring(torrentUrl.LastIndexOf('=') + 1);
+
+                        release.MinimumRatio = 1;
+                        release.MinimumSeedTime = 172800;
+                        release.Title = qLink.Attr("title");
+                        release.Description = qDesc != null ? qDesc.InnerText : release.Title;
+                        release.Guid = new Uri(SiteLink + torrentUrl);
+                        release.Comments = new Uri(release.Guid + "#startcomments");
+
+                        var downloadUrl = $"{SiteLink}download.php?id={torrentId}&rss&passkey={configData.RSSKey.Value}";
+                        release.Link = new Uri(downloadUrl);
+
+                        var qAdded = qRow.Find("#torrent-added").First();
+                        var addedStr = qAdded.Text().Trim();
+                        release.PublishDate = DateTime.ParseExact(addedStr, "dd-MM-yyyyHH:mm:ss", CultureInfo.InvariantCulture);
+
+                        release.Seeders = ParseUtil.CoerceInt(qRow.Find("#torrent-seeders").Text().Trim());
+                        release.Peers = ParseUtil.CoerceInt(qRow.Find("#torrent-leechers").Text().Trim()) + release.Seeders;
+
+                        var sizeStr = qRow.Find("#torrent-size").First().Text();
+                        release.Size = ReleaseInfo.GetBytes(sizeStr);

```

Comment:
The mapping of the release to a category is missing:

 release.Category = MapTrackerCatToNewznab(cat);

I see you have several mappings to a a single category so you might need to do something similar to that in pretome where you map the output to tv rather than tv/sd or tv/hd.